### ISSUE TITLE
Is transitionTo something that ember doesn't recognize

### ISSUE THREAD

**Issue by novakw267** [ga-wdi-boston/capstone-project#581]
I am currently trying to use transitionTo in my edit function to go to the edit form.

The action is getting seen, because I have a console.log that is responding when called. 

components blog-list/index.js
```
    edit() {
      console.log('edit');
      this.transitionTo('blog.edit');
      this.sendAction('edit', this.get('blog'));
    },
```

The action is there at least to call this function, because I get this 
```
Uncaught TypeError: this.transitionTo is not a function
``` 
I have been trying to read the ember docs and see if it was a package I had to download or if it was something that might be part of an old version of ember. 

Overall I am just trying to transition a view state, so the form to enter new values into the blog are visible and the save button is accessible.

**Comment by raq929** [comment]
`transitionTo` is a method on routes. It won't work on components. Move the transition to (haha) the route that catches this action. :)

**Comment by novakw267** [comment]
I saw that right after I posted this. Is there a way to use it to get to a form nested in a component? 

Hold while I will draw something quick and post it. 

In the form a file path basically I am trying to go to something like 'templates/components/blog-list/post" for a file path.

**Comment by raq929** [comment]
No, you need to send the action from the form up to the route.

**Comment by raq929** [comment]
Yay for drawing things!

**Comment by novakw267** [comment]
While drawing that it dawned on me I was wrong about my file pathing and realized I was missing quotes around my editBlog in the template file path, got a new error I will work on that and see where it gets me. 

In addition I think I need to clean up some code haha

```
edit="editBlog"
```
instead of 
```
edit=editBlog
```
I realized that by putting a console log in my transitionTo in my editBlog function in my routes. 
```
  editBlog (blog) {
    console.log('edit2');
    this.transitionTo('blog.edit', blog);
  },
```
When I wasn't getting back I knew it wasn't getting called so I traced the file path back.
